tests/data/AnsysSound_Orders_MultipleParameters_FromAccelWithTacho_2024R2_20241205.txt

```diff
@@ -0,0 +1,808 @@
+AnsysSound_Orders_MultipleParameters 1
+dBSPL
+RPM
+rpm
```

Comment:
why both RPM and rpm?

Reply:
@mattbennett72 One is the parameter name, the other is the unit, which happen to be the same here, whether upper or lower case)